backend/src/api/mutation.rs

```diff
@@ -160,12 +160,16 @@ impl Mutation {
     // and optimistically updating the playlist in Tobira's DB.
     async fn update_playlist(
         id: Id,
-        title: Option<String>,
-        description: Option<String>,
+        metadata: Option<BasicMetadata>,
         entries: Option<Vec<String>>,
         acl: Option<Vec<AclInputEntry>>,
         context: &Context,
     ) -> ApiResult<AuthorizedPlaylist> {
+        let (title, description) = match metadata {
+            Some(m) => (Some(m.title), m.description),
+            None => (None, None),
+        };
```

Comment:
Wait, I don't get why this change was necessary. Commit says:

> Instead of having
> both title and description optional, it
> now uses the `BasicMetadata` struct again,
> which makes sure that the title won't be
> overwritten with a `None` value when
> omitted from the mutation.

But after this code, the title is `None` again, sooo?